cilium/cmd/debuginfo.go

```diff
@@ -106,10 +106,7 @@ func runDebugInfo(cmd *cobra.Command, args []string) {
 }
 
 func printList(w io.Writer, header string, args ...string) {
-	output, err := exec.Command("cilium", args...).CombinedOutput()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error while reading '%s', %s\n", args, err)
-	}
+	output, _ := exec.Command("cilium", args...).CombinedOutput()
```

Comment:
Is it safe to ignore the error?

Reply:
I think so. It was just being logged, if something else is wrong there should be some useful output from Cilium in the `output`. Also IMO this was not adding much value but instead, confused people eyeballing the CI logs for flakes.